modules/administration/pages/troubleshooting/tshoot-hostname-rename.adoc

```diff
@@ -45,45 +45,49 @@ The only mandatory parameter for the script is the newly configured IP address o
 . Reboot the {productname} Server to use the new network configuration and to ensure the hostname has changed.
-. Run the script [command]``spacewalk-hostname-rename`` script with the public IP address of the server.
-    If the server is not using the new hostname, the script fails.
-    Be aware that this script refreshes the pillar data for all {salt} clients: the time it takes to run depends on the number of registered clients.
-
+. On the container host, from the command line, execute the following command to configure the public IP address of the server:
++
+----
+mgrctl exec -ti -- spacewalk-hostname-rename <PUBLIC_IP_ADDRESS>
+----
++
+If the server is not using the new hostname, the script fails.
+  Be aware that this script refreshes the pillar data for all {salt} clients: the time it takes to run depends on the number of registered clients.
 . *Skip this step if the clients are managed via a {productname} proxy.*
   Re-configure the clients directly managed to make them aware of the new hostname and IP address.
-  In the {salt} client configuration file, you must specify the name of the new {salt} master ({productname} Server) (the filename is [path]``/etc/venv-salt-bundle/minion``or, if you do not use the {salt} bundle, [path]``/etc/salt/minion``):
+  In the {salt} client configuration file, you must specify the name of the new {salt} master ({productname} Server).
+  The filename is [path]``/etc/venv-salt-bundle/minion`` or, if you do not use the {salt} bundle, [path]``/etc/salt/minion``:
 +
 ----
 master: <new_hostname>
 ----
 
-. Restart the [systemitem]``venv-salt-minion`` service (if using the {salt} bundle):
+. Restart the server.
+  On the container host, execute the restart command:
 +
 ----
-systemctl restart venv-salt-minion
-----
-
-+
-Or  the [systemitem]``salt-minion``  service (if not using the {salt} bundle):
-+
-----
-systemctl restart salt-minion
+mgradm restart
 ----
 
 . To fully propagate the hostname to the {salt} client configuration apply the high state.
   Applying the high state will update the hostname in the repository URLs.
 
 
 
-[IMPORTANT]
-====
+== Proxy Reconfiguration
+
 Any proxy must be reconfigured.
-The new server certificate and key must be copied to the proxy and the
-[command]``configure-proxy.sh`` script must be run.
-====
+The new server certificate and key must be copied to the proxy.
+For more information, see xref:installation-and-upgrade:install-proxy.adoc[].
+
 
 [NOTE]
 ====
 If you use PXE boot through a proxy, you must check the configuration settings of the proxy.
-On the proxy, run the [command]``configure-tftpsync.sh`` setup script and enter the requested information.
+Only for non-containerized proxies version 4.3, run the [command]``configure-tftpsync.sh`` setup script and enter the requested information.
+In such a case, on the container host, execute:
+
+----
+mgrctl exec -ti -- configure-tftpsync.sh
+----
```

Comment:
This is confusing to me. How about making this `IMPORTANT` instead of `NOTE` and

```suggestion
If you use PXE boot through Non-containerized {productname} Proxy 4.3 [literal]``tftpsync`` needs to be reconfigured.

On the container host, execute:
----
mgrctl exec -ti -- configure-tftpsync.sh
----
```